src/widgets/Markdown/markdown.scss

```diff
@@ -2,7 +2,7 @@
 
 .markdown {
   .form-control {
-    height: 148px !important;
+    resize: vertical;
```

Comment:
Nice feature. Maar IE lijkt het niet te ondersteunen. Check even of dat een harde requirement is. Dan kan je misschien een fallback zetten.

https://developer.mozilla.org/en-US/docs/Web/CSS/resize#Browser_compatibility